### PR TITLE
Add ip exhaustion error to base to show when fields are hidden

### LINKED ISSUE #929: Show a useful error message when IP pool is exhausted
Screen Shot 2022-08-19 at 11 01 04

Document limits as well.

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #929

### CODE REVIEW

File: apps/fz_http/lib/fz_http/devices/device.ex [hunk 1]
```diff
@@ -239,8 +239,8 @@ defmodule FzHttp.Devices.Device do
         else
           add_error(
             changeset,
-            ip_type,
-            "address pool is exhausted. Increase network size or remove some devices."
+            :base,
```
Comment (princemaple) [inline apps/fz_http/lib/fz_http/devices/device.ex:242]: Is this a magic value or something? where does this error get displayed?

Comment (jamilbk) [reply apps/fz_http/lib/fz_http/devices/device.ex:242]: https://github.com/firezone/firezone/blob/fix-ip-exhaustion-error/apps/fz_http/lib/fz_http_web/live/device_live/new_form_component.html.heex#L61

Comment (jamilbk) [reply apps/fz_http/lib/fz_http/devices/device.ex:242]: It's a Rails convention I'm borrowing to display non-field-specific errors

Comment (princemaple) [reply apps/fz_http/lib/fz_http/devices/device.ex:242]: Cool :)

**Review by princemaple — APPROVED** [review]

**Pull request merged**